pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "hatchling",
+    "hatchling>=1.26",
```

Comment:
Is there a way to test that the original issue was fixed in CI?

Perhaps running `pip check` on the installed environment?